tool/tsh/tsh.go

```diff
@@ -2469,3 +2470,15 @@ func handleUnimplementedError(ctx context.Context, perr error, cf CLIConf) error
 	}
 	return trace.WrapWithMessage(perr, errMsgFormat, pr.ServerVersion, teleport.Version)
 }
+
+// printCommand prints command to standard output.
+func printCommand(output io.Writer, cmd *exec.Cmd) {
+	// Add quotes for arguments with '&' to prevent sending the command to
+	// background in terminal.
+	for i, arg := range cmd.Args {
+		if strings.ContainsRune(arg, '&') {
+			cmd.Args[i] = fmt.Sprintf("%q", arg)
```

Comment:
We may want to consider more sophisticated approach to be safe. How about what https://github.com/sergeymakinen/go-quote does?

Reply:
ya, it's definitely a valid concern. Seems quoting behaviour is very inconsistent on windows. I thought about it too but couldn't find a good library. 

https://github.com/sergeymakinen/go-quote seems pretty nice and does what we want. But it has a single maintainer i don't know if we want to depend on that. 

How about
```suggestion
               if strings.ContainsRune(arg, '&') && !strings.ContainsRune(arg, '"') {
			cmd.Args[i] = fmt.Sprintf("%q", arg)
```
so we only quote when it's safe to quote. more complicated case we fall back to original behavior. WDYT

Reply:
(Looks like my old comment above wasn't submitted properly, you may not have seen it. Sorry!)

Anyway, I gave it more thought, I would suggest one of these options:
1. ignore Windows support but explicitly comment on that deficiency, focus on Linux/Mac support  
2. go the extra mile and support Windows either by writing proper code ourselves or leaning on a third-party package (requires vetting it).

The first option is definitely the cheapest and maybe good enough. 

In particular, trying to detect already-quoted commands feels dodgy and fudges the requirements about the shape of the incoming command to be printed.

Reply:
@Tener yes, I was just thinking about treating windows differently. Thanks a lot for ideas. I will close this PR and open a different one once I am ready.